package.json

```diff
@@ -26,8 +26,8 @@
   "homepage": "https://github.com/okonet/metalsmith-webpack-dev-server#readme",
   "dependencies": {
     "chalk": "^1.1.1",
-    "metalsmith": "^1.7.0",
+    "metalsmith": "^2.2.0",
```

Comment:
Looking at this bump I assume there are breaking changes in there. Should this package be concerned about them and also bump a major version?